docs/content/preview/launch-and-manage/monitor-and-alert/metrics/throughput.md

```diff
@@ -12,26 +12,60 @@ menu:
+```json
+{
+  "name": "handler_latency_yb_tserver_TabletServerService_Read",
+  "total_count": 14390,
+  "min": 104,
+  "mean": 171.65,
+  "percentile_75": 0,
+  "percentile_95": 0,
+  "percentile_99": 0,
+  "percentile_99_9": 0,
+  "percentile_99_99": 0,
+  "max": 1655,
+  "total_sum": 2470154
+}
+```
+
+And will include the following attributes:
 
 | Attribute | Description |
 | :--- | :--- |
-| `total_count/count` | The number of times the value of a metric has been measured.
-| `min` | The minimum value of a metric across all measurements.
-| `mean` | The average value of the metric across all measurements.
-| `Percentile_75` | The 75th percentile value of the metric across all measurements.
-| `Percentile_95` | The 95th percentile value of the metric across all measurements.
-| `Percentile_99` | The 99th percentile of the metric across all metrics measurements.
-| `Percentile_99_9` | The 99.9th percentile of the metric across all metrics measurements.
-| `Percentile_99_99` | The 99.99th percentile of the metric across all metrics measurements.
-| `max` | The maximum value of the metric across all measurements.
-| `total_sum/sum` | The aggregate of all the metric values across the measurements reflected in total_count/count.
+| `total_count` | The number of times the latency of a metric has been measured.
+| `min` | The minimum value of the latency for the metric across all measurements.
+| `mean` | The average latency for the metric across all measurements.
+| `Percentile_75` | The latency at which 75% of the requests were completed successfully for the metric.
+| `Percentile_95` | The latency at which 95% of the requests were completed successfully for the metric.
+| `Percentile_99` | The latency at which 99% of the requests were completed successfully for the metric.
+| `Percentile_99_9` | The latency at which 99.9% of the requests were completed successfully for the metric.
+| `Percentile_99_99` | The latency at which 99.99% of the requests were completed successfully for the metric.
+| `max` | The maximum value of latency for the metric across all measurements.
+| `total_sum` | The aggregate latency for the metrics across all measurements.
 
 For example, if `SELECT * FROM table` is executed once and returns 8 rows in 10 microseconds, the `handler_latency_yb_ysqlserver_SQLProcessor_SelectStmt` metric would have the following attribute values: `total_count=1`, `total_sum=10`, `min=10`, `max=10`, and `mean=10`. If the same query is run again and returns in 6 microseconds, then the attributes would be as follows: `total_count=2`, `total_sum=16`, `min=6`, `max=10`, and `mean=8`.
 
-Although these attributes are present in all `handler_latency` metrics, they may not be calculated for all the metrics.
+Although these attributes are present in all latency metrics, they may not be calculated for all the metrics.
```

Comment:
Can you clarify further what you mean by that?

Reply:
Is the next para an example of what you are talking about here?

Reply:
What I mean to say is that although the metrics has attributes like min, mean, p75, p95, p99, etc., they are not calculated by the DB for all metrics. For example:
{
                "name": "handler_latency_yb_master_MasterBackup_CreateSnapshotSchedule",
                "total_count": 1,
                "min": 0,
                "mean": 0.0,
                "percentile_75": 0,
                "percentile_95": 0,
                "percentile_99": 0,
                "percentile_99_9": 0,
                "percentile_99_99": 0,
                "max": 0,
                "total_sum": 16554
            }